crossing-training.lic

```diff
@@ -656,6 +665,19 @@ class CrossingTraining
     end
   end
 
+  def climb_practice
+    check_osrel unless @researching
```

Comment:
I think it would be possible to make a common method from `practice_branch` but making the target and id variable like in `climb_practice`